src/nv_ingest/schemas/text_splitter_schema.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+from pydantic import Field, BaseModel
+
+from typing_extensions import Annotated
+
+
+class TextSplitterSchema(BaseModel):
```

Comment:
Probably want some additional validation so that chunk_overlap has to be < chunk_size.